sklearn/datasets/base.py

```diff
@@ -919,3 +920,25 @@ def _fetch_remote(remote, dirname=None):
                       "file may be corrupted.".format(file_path, checksum,
                                                       remote.checksum))
     return file_path
+
+
+def _refresh_cache(path):
```

Comment:
you put this function in base.py but actually it's designed only for ``fetch_covtype``?